packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -2108,6 +2108,42 @@ describe('plugin-meetings', () => {
         assert.isFunction(locusParser.onDeltaAction);
       });
 
+      it('#updateLocusInfo invokes updateLocusUrl before updateMeetingInfo', () => {
+        const callOrder = [];
+        locusInfo.updateControls = sinon.stub();
```

Comment:
just FYI, in general it's better to use `sinon.stub(locusInfo, 'updateControls')` syntax, because then sinon is able to keep track of all the stubbed methods and can automatically remove the stubs on test teardown. It probably doesn't matter here that much, because `locusInfo` is always recreated in `beforeEach()`, but it's good to follow that approach everywhere in case someone copies the approach to some other test.